tests/pipelines/test_pipelines_common.py

```diff
@@ -116,7 +116,7 @@ def test_vae_tiling(self):
         inputs["return_dict"] = False
         output_2 = pipe(**inputs)[0]
 
-        assert np.abs(output_2 - output_1).max() < 5e-1
+        assert np.abs(to_np(output_2) - to_np(output_1)).max() < 5e-1
```

Comment:
Fixes the VAE error as mentioned in the PR description. This is how it's written in multiple tests but was not done here.